lib/users/middleware/utils.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
Why have specifically middleware utils (and create a sub middleware folder while you're at it) instead of just having user utils?